samples/remote_attestation/README.md

```diff
@@ -125,77 +99,23 @@ The host does the following in this sample:
 
 #### Attesting an Enclave
 
-Attesting an enclave consists of three steps:
+Attesting an enclave consists of two steps:
 
 ##### 1) Generating an Enclave Report
 
-The enclave being attested first needs to generate a cryptographically strong proof of its identity that the challenger can verify. In the sample this is done by asking the SGX platform to generate a `remote report` signed by Intel via the `oe_get_report` method with `OE_REPORT_FLAGS_REMOTE_ATTESTATION` flag. The `remote report` can be verified by the `oe_verify_report` method on a different machine.
-
-An important feature of `oe_get_report` is that you can pass in application specific data as the `reportData` parameter to be signed into the report.
+The enclave being attested first needs to generate a cryptographically strong proof of its identity that the challenger can verify. In the sample this is done by asking the SGX platform to generate a `remote report` signed by Intel via the `oe_get_evidence` method with `OE_FORMAT_UUID_SGX_ECDSA` uuid. The `remote report` can be verified by the `oe_verify_evidence` method on a different machine.
 
-- This is limited to 64 bytes in SGX. As illustrated in the sample, you sign arbitrarily large data into the report by first hashing it and then passing it to the `oe_get_report` method.
-
-- This is useful to bootstrap a secure communication channel between the enclave and the challenger.
-
-  - In this sample, the enclave signs the hash of an ephemeral public key into its report, which the challenger can then use to encrypt a response to it.
-
-  - Other usage examples for `reportData` might be to include a nonce, or to initiate Diffie-Helman key exchange.
 
 ##### 2) Verifying the integrity of the Enclave Report
 
-Once the report is generated and passed to the challenger, the challenger can call `oe_verify_report` to validate the report originated from an Trust Execution Environment (TEE, in the case it's a valid SGX platform).
+Once the report is generated and passed to the challenger, the challenger can call `oe_verify_evidence` to validate the report originated from an Trust Execution Environment (TEE, in the case it's a valid SGX platform).
 
 In the context of Open Enclave on Intel SGX platform, a remote report is verified using the certificate chain issued by Intel which is only valid for SGX platforms.
 
 At this point, the challenger knows that the report originated from an enclave running in a TEE, and that the information in the report can be trusted.
 
 Note that for the Public Preview, remote attestation verification is only supported in the Azure ACC VMs, but Intel will be expanding support for this with Open Enclave SDK more broadly moving forward.
 
-##### 3) Verifying the enclave identity
```

Comment:
This section has to be kept. Instead of oe_identity_t struct, we now use claims:

**
- Ensure that the identity of the enclave matches the expected value:
  - Verify the `unique_id` claim if you want to match the exact bitwise identity of the enclave. Bear in mind that any patches to the enclave will change the `unique_id` claim in the future.
  - Verify the `signer_id` and `product_id` claims if you want to match the identity of an enclave that might span multiple binary versions. This is what the attestation sample does.  
- Ensure that the `security_version` claim of the enclave matches your minimum required security version.
- Ensure that the `custom_claims_buffer` matches the hash of the data provided with the report, as illustrated by the sample.
**

Reply:
See the version I have in PR #3520, will that text work?